lib/tasks/generate_fake_users.rake

```diff
@@ -0,0 +1,23 @@
+require 'faker'
+
+desc 'Generate 10 fake user account if less than 10 users'
+task generate_fake_users: :environment do
+  if User.count < 10
```

Comment:
Maybe use the inverse condition instead and replace with a guard clause? That would avoid some of the indentation further down the line.

Reply:
You cannot use early return here because of the block syntax

Reply:
You can use the `next` keyword to jump to the next block. Will do that